sdks/python/apache_beam/runners/interactive/interactive_beam.py

```diff
@@ -329,6 +359,101 @@ def record(self, pipeline):
     return recording_manager.record_pipeline()
 
 
+class Clusters():
+  """An interface for users to modify the pipelines that are being run by the
+  Interactive Environment.
+
+  Methods of the Interactive Beam Clusters class can be accessed via:
+    interactive_beam.clusters
+
+  Example of calling the Interactive Beam clusters describe method::
+    interactive_beam.clusters.describe()
+  """
+  def __init__(self) -> None:
+    """Instantiates default values for Dataproc cluster interactions.
+    """
+    self._default_cluster_name = 'interactive-beam-cluster' 
+    self._master_urls = bidict()
+    self._dataproc_cluster_managers = {}
+    self._master_urls_to_pipelines = {}
+
+  def describe(self, pipeline: Optional[beam.Pipeline]=None) -> dict:
+    """Returns a description of the cluster associated to the given pipeline.
+
+    If no pipeline is given then this returns a dictionary of descriptions for
+    all pipelines.
+    """
+
+    description = ie.current_env().describe_all_clusters()
+    if pipeline:
+      return description.get(pipeline, None)
+    return description
+
+  @property
+  def default_cluster_name(self) -> str:
+    """The default name to be used when creating Dataproc clusters.
+
+    Defaults to 'interactive-beam-cluster'.
+    """
+    return self._default_cluster_name
+
+  @default_cluster_name.setter
+  def default_cluster_name(self, value: bool) -> None:
+    """Sets the default name to be used when creating Dataproc clusters.
+
+    Defaults to 'interactive-beam-cluster'.
+
+    Example of assigning a default_cluster_name::
+      interactive_beam.clusters.default_cluster_name = 'my-beam-cluster'
+    """
+    self._default_cluster_name = value
+
+  def cleanup(self, pipeline: beam.Pipeline, forcefully=False) -> None:
```

Comment:
s/forcefully/force generally APIs use "force" to indicate this.

Reply:
Noted, thanks!